Classes/common/Encryption/Keychain/CDTEncryptionKeychainManager+Internal.h

```diff
@@ -0,0 +1,47 @@
+//
+//  CDTEncryptionKeychainManager+Internal.h
+//  
+//
+//  Created by Enrique de la Torre Fernandez on 15/04/2015.
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import "CDTEncryptionKeychainManager.h"
+
+/**
+ This category is only for testing purposes. It presents some of the private methods used by
+ CDTEncryptionKeychainManager to complete its job.
+ 
+ Notice that although all these methods are instance methods, none of them modifies the state of a
+ CDTEncryptionKeychainManager instance or writes to disk, however they are easier to override
+ this way.
+ */
+@interface CDTEncryptionKeychainManager (Internal)
+
+- (BOOL)validatedEncryptionKeyData:(CDTEncryptionKeychainData *)data;
```

Comment:
Or possibly, to be a pain, `validateEncryptionKeyData:` as the object is performing a task on `data`, c.f., [For methods that represent actions an object takes, start the name with a verb][1]. `is` should be reserved for accessor methods.

It looks like this method could also be a class method rather than an instance method?

[1]: https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html